core/src/main/java/edu/stanford/futuredata/macrobase/pipeline/CubePipeline.java

```diff
@@ -76,7 +75,7 @@ public Explanation results() throws Exception {
         summarizer.setCountColumn(classifier.getCountColumnName());
         summarizer.setAttributes(attributes);
         summarizer.setMinSupport(minSupport);
-        summarizer.setMinRiskRatio(minRiskRatio);
+        summarizer.setMinRatioMetric(minRatioMetric);
```

Comment:
This sets the kind of "ratio metric" we want to use: can be risk ratio or anything lse